content/authorization/authorizing-api-calls/index.mdx

```diff
@@ -692,33 +692,44 @@ curl "https://accounts.livechat.com/v2/info"
 </Code>
 </Section>
 
-# Customer authorization flow
+# Customer authorization flows
 
 To authorize requests to the [Customer Chat API](/messaging/customer-chat-api/), you'll need a different access token than the one used for the [Agent Chat API](/messaging/agent-chat-api/) and [Configuration API](/management/configuration-api/).
 Throughout the whole API documentation, we'll refer to it as `customer_access_token`.
 
+Customer authorization flows allow you to receive an identity of a customer. You can do it via [Cookie grant](#cookie-grant) if you want to receive an identity of a single customer or [Agent token grant](#agent-token-grant) in order to manage indentities of multiple customers.
+
+## Cookie grant
+
+Cookie grant is based on the `__lc_cid` and `__lc_cst` cookies, which store your customer identity data. These cookies are protected with the `Secure` and `HttpOnly` flags, which means that one can send the cookies only via HTTPS and only specific servers can access them.
+
+The cookies are valid for 2 years, and each next call to `https://accounts.livechat.com/customer/token` extends their lifetime. The `__lc_cid` and `__lc_cst` cookies are necessary to recognize a customer when an access token is being issued.
+
+This flow is recommended for front-end apps that support cookies (like an app running in a web browser). Each user of the app receives an identity if a single customer.
```

Comment:
```suggestion
This flow is recommended for frontend apps that support cookies, such as an app running in a web browser. It has its limitations – all app users will share the same customer identity (the one whose data is stored in cookies).
```

Reply:
I'm not sure if I got the meaning right

Reply:
Not really - I meant that every client (user) of the frontend app will get its own, unique identity and they will be able to issue an access token using this identity. So there will be many customer identities but each of them will be managed separately by its owner

Reply:
I understood something different from our conversation earlier 😅 That Cookie grant limits customer identities to 1 because the browser will not store multiple customers' data at the same time.

PS: What do you mean by owner?

Reply:
In the agent-token flow, owner is the backend integration and it owns many identities. In the cookie flow, owner of the identity is a user of the frontend app, however, there are many users.

Anyway, I have tweaked it a bit. Does it look better, now?

Reply:
Okay, I think I get it finally :D I'll get rid of that info in brackets because it adds more confusion `(the one whose data is stored in cookies)` than value

Reply:
I'll also add one short sentence to Agent token auth that this flow is recommended for backend apps, so that it's straightforward